Python/Product/Cookiecutter/Shared/Interpreters/InterpreterConfiguration.cs

```diff
@@ -98,7 +98,7 @@ public sealed class InterpreterConfiguration {
 
             var cmp = StringComparer.OrdinalIgnoreCase;
             return cmp.Equals(PrefixPath, other.PrefixPath) &&
-                string.Equals(Id, other.Id) &&
+                string.Equals(Id, other.Id, StringComparison.InvariantCultureIgnoreCase) &&
```

Comment:
I'm hesitant here, but the more I think about it - the `Id` may contain a project name or folder name, so I'm thinking we better use `cmp` for the `Id` as well. Same for the duplicate code below.

Reply:
Will be modified in next commit